pex/pex_info.py

```diff
@@ -451,11 +451,18 @@ def raw_pex_root(self):
     @property
     def pex_root(self):
         # type: () -> str
-        pex_root = os.path.expanduser(self.raw_pex_root)
-        if not can_write_dir(pex_root):
+        paths = os.path.expanduser(self.raw_pex_root).split(",")
```

Comment:
I think you probably want to first `split`, then call `os.path.expanduser` on each element?

Reply:
If `/etc/pexrc` (see comment below) is not the answer here, the option really out to be changed to a list option up in `pex/bin/pex.py` and stored as a list here in `PexInfo` too. The, only on read of the value is the list whittled down to one value or a tmp dir if all else fails. There is no reason to court escaping issues when you don't have to in the first place: paths can validly contain commas in them on some systems.

Reply:
Sure. Let me try this way.

Reply:
If you mean the environment variable (`Variables.PEX_ROOT`), I think that can remain singular. If you're exporting an environment variable you're exporting it on a given host. If that host has a disk full condition, you'd just change that environment variable manually. This is what you can do via `/etc/pexrc`.

When you think about the details here - unless I'm missing something, a disk can go full at any time, even in between when Pex 1st checks for the condition and later when it goes to write files during a single run of a PEX. You can never fully avoid that race condition. You can check early in a PEX run whether a disk is _already_ full - which I think is what you're trying to do - but that's still best effort if the answer is no. Now as far as getting an answer yes - the disk is full - how do you propose that is done? I have no clue how you do this except by something hacky like: assume the system has the `df` binary on the path and use it to get the current full state of the disks on the system and then map the given pex root to one of those disks by checking the mount prefix that contains it. That seems super hacky. Do you have some other idea?

Reply:
How about adding a variable like `PEX_ROOT_FALLBACK` that we can fall back to on exceptions?

Reply:
The problem is the pex root is accessed all over the place in code. You'd need to wrap every one of those in a try / except. Like I said, the condition of a disk is racy. It can go bad or full at any time, even in the middle of a Pex run.